client/packages/invoices/src/OutboundShipment/DetailView/OutboundLineEdit/Allocation.tsx

```diff
@@ -42,12 +45,13 @@ export const Allocation = ({
   );
 
   useEffect(() => {
-    initialise(
-      itemData,
-      AllocationStrategy.FEFO,
+    initialise({
+      input: itemData,
+      strategy: AllocationStrategy.FEFO,
       allowPlaceholder,
-      scannedBatch
-    );
+      allocateVaccineItemsInDoses,
```

Comment:
When initialising allocation context, pass through the store pref. This will set the context's `allocateIn` value. From there, everything should check the context for what to display/do, rather than lots of pref/isVaccineItem checks everywhere